src/cgi/utils.cpp

```diff
@@ -0,0 +1,60 @@
+#include "../../inc/cgi.hpp"
+#include <iostream>
+#include <map>
+#include <string>
+
+namespace utils
+{
+void
+print_container_map(std::map<std::string, std::string> myMap)
+{
+	std::map<std::string, std::string>::iterator it;
+	for (it = myMap.begin(); it != myMap.end(); ++it)
```

Comment:
Why not merge these two lines ?
```suggestion
	for (std::map<std::string, std::string>::iterator it = myMap.begin(); it != myMap.end(); ++it)
```